src/packages/utils/src/utils/signature.ts

```diff
@@ -0,0 +1,37 @@
+import secp256k1 from "@ganache/secp256k1";
+
+type ECDSASignOutput = {
+  signature: Uint8Array;
+  recid: number;
+};
+
+export function ecsign(msgHash: Uint8Array, privateKey: Uint8Array) {
```

Comment:
Could potentially define (and export) a `Signature` return type for this function, which you could use to avoid defining the type online in `legacy-transaction.ts` 🤷